frontend/src/core/entities/actions.js

```diff
@@ -37,6 +39,19 @@ export function receive(entities: Entities, hasMore: boolean): ReceiveAction {
         hasMore,
     };
 }
+/**
+ * Update the siblings of an entity.
+ */
+export type ReceiveSiblingsAction = {
+    type: typeof RECEIVE_SIBLINGS,
+    entities: Entities,
+};
+export function receiveSiblings(entities: Entities): ReceiveSiblingsAction {
```

Comment:
`Entities` type here and in line 47 needs to be replaced with a new flow type that will match the backend response. You can define it in `core/api/types`.

Something like this:
```
export type EntitySiblings = {|
    +preceding: Array<Entity>;,
    +succeeding: Array<Entity>;,
|};
```